main.go

```diff
@@ -217,10 +218,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	tokenLifetimeInHours, err := util.ReadEnvVar("ONBOARDING_TOKEN_LIFETIME", 48, strconv.Atoi)
```

Comment:
The context here is not specific to onboarding?
```suggestion
	onboardingTokenLifetimeInHours, err := util.ReadEnvVar("ONBOARDING_TOKEN_LIFETIME", 48, strconv.Atoi)
```

Reply:
done.